bucket/database.net.json

```diff
@@ -0,0 +1,25 @@
+{
+    "homepage": "https://fishcodelib.com/Database.htm",
+    "description": "Database .NET is an innovative, powerful and intuitive multiple database management tool.",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://fishcodelib.com/EULA.htm"
+    },
+    "version": "26.7.6967.1",
+    "hash": "0cea4b8b18469fb0b3b7830acb680d8e1addbc6f0f4f32a4fd53dac616c30f11",
+    "url": "https://fishcodelib.com/files/DatabaseNet4.zip",
+    "shortcuts": [
+        [
+            "Database4.exe",
+            "Database.NET"
+        ]
+    ],
+    "persist": "Database_Files",
+    "checkver": {
+        "url": "https://fishcodelib.com/version.xml",
+        "re": "<ver name=\"database4\" md5=\"[a-zA-Z0-9]+\" ver=\"([\\d.]+)\""
+    },
+    "autoupdate": {
+        "url": "https://fishcodelib.com/files/DatabaseNet4.zip"
```

Comment:
```suggestion
        "url": "https://fishcodelib.com/files/DatabaseNet4.zip",
        "hash": {
            "url": "https://fishcodelib.com/version.xml",
            "regex": "database4\"\\s*md5=\"([A-Za-z\\d]{32})\""
        }
```

Reply:
It's one time bug or temporary "solution"?

Reply:
I don't know. But now all md5 in the xml is incorrect...
So maybe we shouldn't use it.

Reply:
There is no problem to have it. If it's one time issue next update will be fine and extracted automatically. If it's persist, hash extraction will fail and file will be downloaded (No problem) and developers could be contacted about this.

Reply:
Hm, I don't think this is a one time issue:

- database5pro is missing a zero at position 1
- database4 is missing a zero at position 13
- dbmigration is correct
- dictionary is wrong
- convert is missing a zero at position 15

Maybe this should report this to fish.